teams/wg-ffi-unwind.toml

```diff
@@ -20,3 +26,9 @@ zulip-stream = "project-ffi-unwind"
 
 [[github]]
 orgs = ["rust-lang"]
+
+[[zulip-groups]]
+name = "WG-ffi-unwind"
+
+# Katelynn Martin has two Zulip accounts.
```

Comment:
```suggestion
# Katelyn Martin has two Zulip accounts.
```

Interesting. Are both of these active on an ongoing basis?

Reply:
I hadn't checked, actually, but it looks like one account was effectively unused, so I've removed that one.